test/test_torch.py

```diff
@@ -10922,31 +10922,52 @@ def test_masked_scatter_bool_tensor(self, device):
         self.assertEqual(dst, torch.tensor([True, True, True], device=device))
 
     def test_masked_select(self, device):
-        warn = 'masked_select received a mask with dtype torch.uint8,'
+        if device == 'cpu':
+            warn = 'masked_select received a mask with dtype torch.uint8,'
+        else:
+            warn = 'indexing with dtype torch.uint8 is now deprecated, pl'
         for dt in torch.testing.get_all_dtypes():
```

Comment:
you can use @dtypes(*torch.testing.get_all_dtypes()) decorator instead of loop

Reply:
Done